float32 lat
float32 lon
float32 z
string frame_id
bool frame_locked
---
bool success
string message
